environment_linux_cpuonly.yml

```diff
@@ -44,11 +44,12 @@ dependencies:
   # testing
   - parameterized
   - pip:
-    - open3d
-    - opencv-python>=4.5.4.58
-    - pydegensac
-    - colour
-    - pycolmap>=0.1.0
-    - trimesh[easy]
-    - gtsam==4.2a8
-    - pydot
+      - open3d
```

Comment:
why the indentation change?

Reply:
Done automatically by the Black formatter (see [CONTRIBUTING.md](https://github.com/borglab/gtsfm/blob/master/CONTRIBUTING.md))